src/protocol.cpp

```diff
@@ -199,3 +199,127 @@ const std::vector<std::string> &getAllNetMessageTypes()
 {
     return allNetMessageTypesVec;
 }
+
+
+uint8_t GetShortCommandIDFromCommand(const std::string cmd)
+{
+    if (cmd == NetMsgType::ADDR) {
+        return NetMsgType::ADDR_SHORT_ID;
+    } else if (cmd == NetMsgType::BLOCK) {
+        return NetMsgType::BLOCK_SHORT_ID;
+    } else if (cmd == NetMsgType::BLOCKTXN) {
+        return NetMsgType::BLOCKTXN_SHORT_ID;
+    } else if (cmd == NetMsgType::CMPCTBLOCK) {
+        return NetMsgType::CMPCTBLOCK_SHORT_ID;
+        ;
+    } else if (cmd == NetMsgType::FEEFILTER) {
+        return NetMsgType::FEEFILTER_SHORT_ID;
+    } else if (cmd == NetMsgType::FILTERADD) {
+        return NetMsgType::FILTERADD_SHORT_ID;
+    } else if (cmd == NetMsgType::FILTERCLEAR) {
+        return NetMsgType::FILTERCLEAR_SHORT_ID;
+        ;
+    } else if (cmd == NetMsgType::FILTERLOAD) {
+        return NetMsgType::FILTERLOAD_SHORT_ID;
+    } else if (cmd == NetMsgType::GETADDR) {
+        return NetMsgType::GETADDR_SHORT_ID;
+    } else if (cmd == NetMsgType::GETBLOCKS) {
+        return NetMsgType::GETBLOCKS_SHORT_ID;
+        ;
+    } else if (cmd == NetMsgType::GETBLOCKTXN) {
+        return NetMsgType::GETBLOCKTXN_SHORT_ID;
+    } else if (cmd == NetMsgType::GETDATA) {
+        return NetMsgType::GETDATA_SHORT_ID;
+    } else if (cmd == NetMsgType::GETHEADERS) {
+        return NetMsgType::GETHEADERS_SHORT_ID;
+        ;
+    } else if (cmd == NetMsgType::HEADERS) {
+        return NetMsgType::HEADERS_SHORT_ID;
+    } else if (cmd == NetMsgType::INV) {
+        return NetMsgType::INV_SHORT_ID;
+    } else if (cmd == NetMsgType::MEMPOOL) {
+        return NetMsgType::MEMPOOL_SHORT_ID;
+        ;
+    } else if (cmd == NetMsgType::MERKLEBLOCK) {
+        return NetMsgType::MERKLEBLOCK_SHORT_ID;
+    } else if (cmd == NetMsgType::NOTFOUND) {
+        return NetMsgType::NOTFOUND_SHORT_ID;
+    } else if (cmd == NetMsgType::PING) {
+        return NetMsgType::PING_SHORT_ID;
+        ;
+    } else if (cmd == NetMsgType::PONG) {
+        return NetMsgType::PONG_SHORT_ID;
+    } else if (cmd == NetMsgType::SENDCMPCT) {
+        return NetMsgType::SENDCMPCT_SHORT_ID;
+        ;
+    } else if (cmd == NetMsgType::SENDHEADERS) {
+        return NetMsgType::SENDHEADERS_SHORT_ID;
+        ;
+    } else if (cmd == NetMsgType::TX) {
+        return NetMsgType::TX_SHORT_ID;
+    } else if (cmd == NetMsgType::VERACK) {
+        return NetMsgType::VERACK_SHORT_ID;
+    } else if (cmd == NetMsgType::VERSION) {
+        return NetMsgType::VERSION_SHORT_ID;
+        ;
+    }
+    return 0; //no short command
+}
+
+bool GetCommandFromShortCommandID(uint8_t shortID, std::string& cmd)
```

Comment:
Please replace occurrences of "command" with something like "msg_type" at least in new code. See also #18533

Reply:
I'd like to keep it to be consistent with BIP324 (and other bips). Changing it might make it more confusing.

Reply:
I see @MarcoFalke's point. A P2P protocol has no commands, just messages and message ids. But yea it's unfortunate that was not caught in the BIP.